debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/gccheck/CheckBase.java

```diff
@@ -79,7 +79,8 @@
 	public static final int J9MODRON_GCCHK_MISC_ALWAYS_DUMP_STACK = 0x00004000;
 	public static final int J9MODRON_GCCHK_MISC_DARKMATTER = 0x00008000;
 	public static final int J9MODRON_GCCHK_MISC_MIDSCAVENGE = 0x00010000;
-	public static final int J9MODRON_GCCHK_MISC_OWNABLESYNCHRONIZER_CONSISTENCY = 0x00020000;
+	public static final int J9MODRON_GCCHK_VALID_INDEXABLE_DATA_ADDRESS = 0x00020000;
+	public static final int J9MODRON_GCCHK_MISC_OWNABLESYNCHRONIZER_CONSISTENCY = 0x00040000;
```

Comment:
None of the constants `J9MODRON_GCCHK_*` should be defined here. They should be accessed from DDR-generated code derived from `CheckBase.hpp` to avoid the duplication and potential disagreement.

The value of `J9MODRON_GCCHK_MISC_OWNABLESYNCHRONIZER_CONSISTENCY` is now wrong for old core files. These will need to be made available to DDR and listed in `CompatibilityConstants29.dat`.

Reply:
I believe this is an exceptional case when constants are used for internal GC Check purpose only (run time flag to request check feature but not other JVM functionality). They might be or not to be the same between DDR GC Check vs. Runtime GC Check. Also I think there is no backwards compatibility issue for `J9MODRON_GCCHK_MISC_OWNABLESYNCHRONIZER_CONSISTENCY` as far as it is used run time only (Am I missing something?)
Currently we keep constants for DDR GC Check and Runtime GC Check synchronized but there is no functional dependency between them.
So if in general I agree with statement we should not allow duplication for definitions in this particular case I am not sure we want to create dependency between DDR and Runtime GC Checks
@amicic What do you think?

Reply:
Ok I see. Should I make this change for all `J9MODRON_GCCHK_*` constants and update DDR code for all these constants accordingly?

Reply:
As long as those constants don't represent anything that might exist in a core file, they don't need to be stable (or even match the native code values). That the names are the same makes the answer unclear.

Reply:
Since they are not part of core files and don't impose compatibility issue, I would not add new changes to sync them in this PR. But, in general, it would be nice they are synced, if someone finds time/will to do the work.

Reply:
>Ok I see. Should I make this change for all J9MODRON_GCCHK_* constants and update DDR code for all these constants accordingly?

I think this place is ok "as is" and no further changes required for this PR. If we decided to do cleaning/renaming/refactoring it can be addressed in different request

Reply:
On the other hand, there's no need to change the values used in native code, so the existing names would remain in sync.